cmrtlib/linux/hardware/drm_device.h

```diff
@@ -950,6 +951,7 @@ static int drmParseSubsystemType(int maj, int min)
     snprintf(path, PATH_MAX, "/sys/dev/char/%d:%d/device/subsystem",
         maj, min);
 
+    memset(link, 0, sizeof(link));
```

Comment:
link should be initialized by line 937？
https://github.com/intel/media-driver/pull/1508/files#diff-4ef24d4d6996a3f7d92a338cfee90f859972182c715f2adfeb4e3605ece1a722R937